launch_gz/launch_gz/actions/gzserver.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Module for the GzServer action."""
+
+from typing import List
+from typing import Optional
+
+from launch.action import Action
+from launch.actions import IncludeLaunchDescription
+from launch.frontend import expose_action, Entity, Parser
+from launch.launch_context import LaunchContext
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.some_substitutions_type import SomeSubstitutionsType
+from launch.substitutions import PathJoinSubstitution
+from launch_ros.substitutions import FindPackageShare
+
+@expose_action('gzserver')
+class GzServer(Action):
+	"""Action that executes a gzserver ROS [composable] node."""
+
+	def __init__(
+		self,
+		*,
+		world_sdf_file: SomeSubstitutionsType,
+		world_sdf_string: SomeSubstitutionsType,
+		container_name: SomeSubstitutionsType,
+		use_composition: SomeSubstitutionsType,
```

Comment:
All of these parameters are optional in `gzserver.launch.py` right? Can we do the same here?

Here's an example: https://github.com/ros2/launch_ros/blob/2bf4e6057dea57669c19395f6f39d390bd420ee7/launch_ros/launch_ros/actions/node.py#L121-L129

Reply:
Done in [01345e2](https://github.com/gazebosim/ros_gz/pull/528/commits/01345e2d95d76998f3ab8fbd341e2164d075a009).